scripts/tasks/src/postprocess-amd.ts

```diff
@@ -9,8 +9,12 @@ export async function postprocessAmdTask() {
   const ts = await import('typescript');
 
   mod('lib-amd/**/*.js').asTypescript((node, modder) => {
-    if (ts.isIfStatement(node)) {
```

Comment:
mismatch between ts in `riceburn` and ts, it's quite outdated lib (riceburn is deprecated, it's analogue codemad last updated 4 years ago), not sure if we still need this